### PR TITLE
docs: refine installation of iomesh

### PR DESCRIPTION
**Pull request by WqVoon**
请直接通过访问 http://192.168.31.213:3000/next/deploy/install-iomesh#install-iomesh 进行 review

### CODE REVIEW

File: docs/deploy/install-iomesh.md [hunk 1]
```diff
@@ -224,116 +180,40 @@ helm repo add iomesh http://iomesh.com/charts
...
+    iomesh-zookeeper-0                                     1/1     Running   0          3m14s
+    iomesh-zookeeper-1                                     1/1     Running   0          2m59s
+    iomesh-zookeeper-2                                     1/1     Running   0          2m20s
+    iomesh-zookeeper-operator-7b5f4b98dc-fxfb6             1/1     Running   0          3m20s
+    operator-85877979-5fvvn                                1/1     Running   0          3m20s
+    operator-85877979-74rl6                                1/1     Running   0          3m20s
+    operator-85877979-cvgcz                                1/1     Running   0          3m20s
     ```
 
 IOMeshCluster is now installed successfully, please go to [Setup IOMesh](setup-iomesh.md).
```
Comment (nanana-w) [inline docs/deploy/install-iomesh.md:219]: IOMeshCluster --> IOMesh Cluster?

Comment (WqVoon) [reply docs/deploy/install-iomesh.md:219]: @gxglls 看下👀？

Comment (gxglls) [reply docs/deploy/install-iomesh.md:219]: ok，用 IOMesh Cluster 好一些

**Comment by jeffwubj** [comment]
may you share the motivation of not using iomesh.run/install_iomesh_el7.sh ? one single command to install everything seems to be a good solution for me

**Review by jeffwubj — COMMENTED** [review]
May you share the motivation of not using iomesh.run/install_iomesh_el7.sh ? one single command to install everything seems good for me.

PX do not have such script but all PX installation needs is kubectl.

**Comment by WqVoon** [comment]
> May you share the motivation of not using iomesh.run/install_iomesh_el7.sh ? one single command to install everything seems good for me.
> 
> PX do not have such script but all PX installation needs is kubectl.

这个似乎是因为阿里那边提的需求，希望减少 helm chart 的数量方便在一个 values.yaml 里做一些改动？@gxglls 麻烦帮看下

**Comment by gxglls** [comment]
> > May you share the motivation of not using iomesh.run/install_iomesh_el7.sh ? one single command to install everything seems good for me.
> > PX do not have such script but all PX installation needs is kubectl.
> 
> 这个似乎是之前外面提的需求，希望减少 helm chart 的数量方便在一个 values.yaml 里做一些改动？@gxglls 麻烦帮看下

脚本和手动安装的方式都提供，hongyu 这个 pr 仅仅是把原来手动安装步骤中安装 3 个 charts 的步骤改为直接安装一个 all-in-one chart。iomesh.run/install_iomesh_el7.sh 的逻辑同时也会改为只安装一个 all-in-one chart

File: docs/deploy/install-iomesh.md [hunk 2]
```diff
@@ -224,119 +180,43 @@ helm repo add iomesh http://iomesh.com/charts
...
+    iomesh-zookeeper-0                                     1/1     Running   0          3m14s
+    iomesh-zookeeper-1                                     1/1     Running   0          2m59s
+    iomesh-zookeeper-2                                     1/1     Running   0          2m20s
+    iomesh-zookeeper-operator-7b5f4b98dc-fxfb6             1/1     Running   0          3m20s
+    operator-85877979-5fvvn                                1/1     Running   0          3m20s
+    operator-85877979-74rl6                                1/1     Running   0          3m20s
+    operator-85877979-cvgcz                                1/1     Running   0          3m20s
+    ```
+
+IOMesh Cluster is now installed successfully, please go to [Setup IOMesh](setup-iomesh.md).
```
Comment (yuqiu123) [inline docs/deploy/install-iomesh.md:219]: successfully逗号改为句点，please首字母大写，改为Please。

**Comment by jeffwubj** [comment]
> > > May you share the motivation of not using iomesh.run/install_iomesh_el7.sh ? one single command to install everything seems good for me.
> > > PX do not have such script but all PX installation needs is kubectl.
> > 
> > 
> > 这个似乎是之前外面提的需求，希望减少 helm chart 的数量方便在一个 values.yaml 里做一些改动？@gxglls 麻烦帮看下
> 
> 脚本和手动安装的方式都提供，hongyu 这个 pr 仅仅是把原来手动安装步骤中安装 3 个 charts 的步骤改为直接安装一个 all-in-one chart。iomesh.run/install_iomesh_el7.sh 的逻辑同时也会改为只安装一个 all-in-one chart

那应该在安装文档里保留 Shell 的安装方式 ?

**Comment by WqVoon** [comment]
> > > > May you share the motivation of not using iomesh.run/install_iomesh_el7.sh ? one single command to install everything seems good for me.
> > > > PX do not have such script but all PX installation needs is kubectl.
> > > 
> > > 
> > > 这个似乎是之前外面提的需求，希望减少 helm chart 的数量方便在一个 values.yaml 里做一些改动？@gxglls 麻烦帮看下
> > 
> > 
> > 脚本和手动安装的方式都提供，hongyu 这个 pr 仅仅是把原来手动安装步骤中安装 3 个 charts 的步骤改为直接安装一个 all-in-one chart。iomesh.run/install_iomesh_el7.sh 的逻辑同时也会改为只安装一个 all-in-one chart
> 
> 那应该在安装文档里保留 Shell 的安装方式 ?

是说保留 iomesh.run/install_iomesh_el7.sh 的脚本吗？这个是有保留的，放在安装文档的最上面

**Review by kylezh — APPROVED** [review]

**Pull request merged**